package.json

```diff
@@ -25,20 +25,26 @@
   },
   "homepage": "https://github.com/Lemoncode/simplechart#readme",
   "dependencies": {
+    "@zeit/next-css": "^0.2.0",
     "@material-ui/core": "^3.0.1",
     "@zeit/next-sass": "^0.2.0",
     "@zeit/next-typescript": "^1.1.0",
     "babel-polyfill": "^6.26.0",
+    "css-loader": "^1.0.0",
     "d3": "^5.6.0",
     "d3-composite-projections": "^1.2.0",
     "express": "^4.16.3",
     "isomorphic-unfetch": "^2.1.1",
     "next": "^6.1.1",
+    "postcss-loader": "^3.0.0",
     "qs": "^6.5.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-tag-input": "^5.1.2",
```

Comment:
Material ui has [`chips`](https://material-ui.com/demos/chips/) I think we don't need install other dependency. What do you think?